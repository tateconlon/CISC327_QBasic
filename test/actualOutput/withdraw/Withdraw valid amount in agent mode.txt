login as agent successful
Account 1234567 does not exist
transaction code 500000 invalid
Account 1234567 does not exist
transaction code 499999 invalid
Transaction summary written and system logged out successfully