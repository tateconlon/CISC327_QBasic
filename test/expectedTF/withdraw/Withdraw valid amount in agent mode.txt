DEP 1234567 500000 *** ***
WDR 499999 *** ***
EOS