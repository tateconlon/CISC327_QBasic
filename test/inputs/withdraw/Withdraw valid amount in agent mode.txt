login
agent
deposit
1234567
500000
withdraw
1234567
499999
logout