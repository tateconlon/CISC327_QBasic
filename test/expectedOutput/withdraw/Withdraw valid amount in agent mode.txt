login as agent successful
deposit 1234567 500000 successful
withdraw 1234567 499999 successful
Transaction summary written and system logged out successfully